projects/benqi-lending/__mocks__/@heyanon/sdk.ts

```diff
@@ -0,0 +1,20 @@
+import { vi } from 'vitest';
+
+export const toResult = vi.fn().mockImplementation((data: string, isError: boolean) => ({ data, success: !isError }));
+
+export const getChainFromName = vi.fn().mockImplementation((chainName: string) => {
+    switch (chainName) {
+        case 'Avalanche':
+            return 43114;
+        case 'Ethereum':
+            return 1;
```

Comment:
Again -> probably Eth will not be needed

Reply:
See https://github.com/RealWagmi/anon-integration-guide/pull/16#discussion_r1935200021